docs/manual/developer/06_contributing_with_content.md

```diff
@@ -395,6 +387,8 @@ specified, and they depend on each other. Macros that begin with
 make sure that OCIL and OCIL clauses are defined and consistent. Macros
 that begin with underscores are not meant to be used in descriptions.
 
+You can check documentation for all macros in the [Jinja Macros Reference section](../../index.html#jinja-macros-reference).
```

Comment:
Unfortunately this type of reference doesn't work when the content is build by readthedocs and I was not able to find a workaround for that similar as the deleted line 388

Reply:
It seems a lot of references in https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html with the same syntax are broken.

Reply:
I have rewritten the paragraph with the broken reference.

Reply:
> It seems a lot of references in https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html with the same syntax are broken.

these come when we did the conversion from .adoc to .md and when we hosted in github. Those we can easily fix by changing to the actual file in the github project.